pkg/sensors/program/loader_windows.go

```diff
@@ -0,0 +1,281 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package program
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/link"
+	"github.com/cilium/tetragon/pkg/bpf"
+	"github.com/cilium/tetragon/pkg/logger"
+	"github.com/cilium/tetragon/pkg/sensors/unloader"
+	"golang.org/x/sys/windows"
+)
+
+var (
+	notSupportedWinErr     = errors.New("not supported on windows")
+	programTypeProcessGUID = makeGUID(0x22ea7b37, 0x1043, 0x4d0d, [8]byte{0xb6, 0x0d, 0xca, 0xfa, 0x1c, 0x7b, 0x63, 0x8e})
+	attachTypeProcessGUID  = makeGUID(0x66e20687, 0x9805, 0x4458, [8]byte{0xa0, 0xdb, 0x38, 0xe2, 0x20, 0xd3, 0x16, 0x85})
+)
+
+func makeGUID(data1 uint32, data2 uint16, data3 uint16, data4 [8]byte) windows.GUID {
+	return windows.GUID{Data1: data1, Data2: data2, Data3: data3, Data4: data4}
+}
+
+func WinAttachStub(_ *ebpf.Collection, _ *ebpf.CollectionSpec,
```

Comment:
could be local, also maybe more decriptive name like windowsAttachNotSupp ?

Reply:
Adding Not Supported to a function name seems little off. I think  the word 'stub' should communicate the meaning.